KOGITO-4541 Move quickstart code as extended usecases to current spri… (#3500)
kiegroup/kie-docs: doc-content/kogito-docs/src/main/asciidoc/release-notes/chap-kogito-release-notes.adoc

For more information about the add-on configuration, see xref:con-bpmn-process-svg-addon_kogito-developing-process-services[_{PRODUCT} process SVG add-on_].

==== New example applications in `kogito-examples` repository

> You can now use the following example applications in the `kogito-examples` repository in Github to interact with a process in springboot using the {PRODUCT} Management and Task Console:

Reviewer: ```suggestion
You can now use the following example applications in the `kogito-examples` repository in Github to interact with a process in Spring Boot using the {PRODUCT} Management and Task Console:
```

> * https://github.com/kiegroup/kogito-examples/tree/stable/process-usertasks-springboot-with-console[process-usertasks-springboot-with-console]: Enables you to explore a user task process using the Management and Task Console and provides an easy way to create the required services using docker-compose including Kafka, Infinispan, Data Index, Management Console, and Task Console. Also, this example application shows the process interaction using the Management and Task console.

Reviewer: ```suggestion
* https://github.com/kiegroup/kogito-examples/tree/stable/process-usertasks-springboot-with-console[process-usertasks-springboot-with-console]: Enables you to explore a user task process using the Management and Task Console, and provides an easy way to create the required services using docker-compose including Kafka, Infinispan, Data Index, Management Console, and Task Console. Also, this example application shows the process interaction using the Management and Task console.
```

> * https://github.com/kiegroup/kogito-examples/tree/stable/process-usertasks-with-security-oidc-springboot-with-console[process-usertasks-with-security-oidc-springboot-with-console]: Shows the interaction of the Management and Task console when security is enabled using an open ID connector such as Keycloak.

Reviewer: ```suggestion
* https://github.com/kiegroup/kogito-examples/tree/stable/process-usertasks-with-security-oidc-springboot-with-console[process-usertasks-with-security-oidc-springboot-with-console]: Shows the interaction of the Management and Task console when security is enabled using an open ID connector, such as Keycloak.
```
